planetary-social/nos
# #1596: Public Key screen in onboarding

## Issues covered
#1596 

## Description
Adds the Public Key screen to the onboarding.

## How to test
1. Navigate to Settings
2. Enable New Onboarding Flow
3. Log out
4. Flow through the new onboarding
5. Observe new Public Key screen

## Screenshots/Video

https://github.com/user-attachments/assets/d7bb7cae-005e-4325-8703-0d0d469694f0
